### DISCUSSION TITLE
Deleting slurm cluster in the case terraform output is lost

### DISCUSSION THREAD

**Discussion by fdmalone** [GoogleCloudPlatform/cluster-toolkit#3415] [General]
Is there a script somewhere for replicating the `gcluster destroy ...` command in the case that the terraform output is lost?

**Comment by ighosh98** [comment]
Could you explain what you mean by terraform output? Are you referring to the folder of terraform state files that when you run `gcluster create`? If yes, if you have deleted the folder and haven't stored a copy of the state elsewhere(either on local or gcs), you would not be able to run the destroy command.

**Comment by fdmalone** [comment]
yes exactly, I was wondering if a hacky script already existed for replicating the resource deletion in the event of the loss of the terraform state / output of `gcluster create`.

**Reply by ighosh98** [reply]
We don't have this as of now I think. But I will check again with the team and update the thread.
